src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -578,28 +407,22 @@ protected int getCountOfNonRefEvents(final ReadPileup pileup, final byte refBase
     }
 
     private double calculateF(final ReadPileup pileup, final byte refBase, final byte minBaseQual) {
```

Comment:
I really wanted to stream this but both the total and alt counts require filtering by base quality and `isDeletion`.  So we could either have two streams with duplicate filtering (inelegant), or `collect` a common filtered stream to an intermediate list, which we would then stream again (also inelegant).  I could also imagine something with `MutableInt`s but I couldn't think of anything that beat the original `for` loop.